metrics2/metrics2/src/main/java/io/helidon/metrics/ExponentiallyDecayingReservoir.java

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
This class is exact duplicate of the class in `metrics`. Probably should be extracted to common metrics?

Reply:
Recall that the goal, eventually, is to _not_ have to support both versions of metrics (1.1 and 2.0) at the same time from a single copy of our internal metrics clients (accomplished in this PR using the bridge). If and when that day comes, we want to be able to remove easily the files tied to the older version -- the entire `metrics` and `microprofile/metrics` modules and even `common/metrics`, leaving only what are now called `metrics2` and `microprofile/metrics2`. 

For that reason, the PR has avoided optimizing the copy by refactoring identical copies into `common/metrics` as that would complicate the changes later to go back to a single copy of metrics and microprofile/metrics.

One cost is, of course, the possible need to update some code in two places if we find bugs. That is a price of making easier the later removal of the second copy.

Reply:
Yes, you are right - we want to remove metrics 1.x and all related additional modules once we move to 2.0, so this makes sense